Testscripts/Linux/utils.sh

```diff
@@ -1963,23 +1963,52 @@ function get_host_version() {
 
 # Validate the exit status of previous execution
 function check_exit_status() {
+	# The failed/aborted options are used when Linux script is the testscript,
+	# to check the checkpoint and set test result.
 	exit_status=$?
 	message=$1
+	test_state=$2
+	failed_message=$3
 
 	cmd="echo"
 	if [ $exit_status -ne 0 ]; then
 		$cmd "$message: Failed (exit code: $exit_status)"
 		UpdateSummary "$message Failed·(exit·code:·$exit_status)"
-		if [ "$2" == "exit" ]; then
-			SetTestStateAborted
-			exit $exit_status
-		fi
+		case "${test_state}" in
+			failed)
+				SetTestStateFailed
+				LogErr "$failed_message"
+				exit 0
+				;;
+			aborted)
+				SetTestStateAborted
+				exit 0
+				;;
+			exit)
+				SetTestStateAborted
+				exit $exit_status
+				;;
+			*)
+				LogWarn "Unsupported check_exit_status option: ${test_state}"
+				;;
+		esac
 	else
 		$cmd "$message: Success"
 		UpdateSummary "$message: Success"
 	fi
 }
 
+# Validate the previous command exit code is 0
+function CheckPass() {
+	check_exit_status "$1" "failed" "$2"
```

Comment:
failed?

Reply:
Hi @LiliDeng,
The "failed" here is an option of "check_exit_status", which means if not match the status set test state as "TestFailed".